bundler/lib/bundler/templates/newgem/ext/newgem/extconf-c.rb.tt

```diff
@@ -2,4 +2,6 @@
 
 require "mkmf"
 
+$CFLAGS << " -fvisibility=hidden "
```

Comment:
Would it make sense to add small comment in here explaining why this is useful?

Reply:
Uh, I thought I did, I must have had a brainfart. I'll add a comment.